src/Microsoft.Diagnostics.Monitoring.HostingStartup/ParameterCapturing/ParameterCapturingStrings.resx

```diff
@@ -147,6 +147,9 @@
   <data name="UnknownArgumentValue" xml:space="preserve">
     <value>unknown</value>
   </data>
+  <data name="UnknownDeclaringType" xml:space="preserve">
+    <value>&lt;unknown_type&gt;</value>
```

Comment:
I know that this code is somewhat inconsistent on this, but I don't think this should be a localizable resource. This type of information is likely to be machine parsable, so it should use well-known constants.